addons/cni-cilium/cilium.yaml

```diff
@@ -1,5 +1,12 @@
 {{ $kubernetesVersion := semver .Config.Versions.Kubernetes }}
 apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/part-of: cilium
+  name: kube-system
```

Comment:
Do we really want the Cilium addon to manage the `kube-system` namespace? That sounds potentially disastrous 😅